utils/gyb_syntax_support/ExprNodes.py

```diff
@@ -569,4 +569,19 @@
              Child('Arguments', kind='FunctionCallArgumentList'),
              Child('RightParen', kind='RightParenToken'),
          ]),
+
+    # e.g. #assert(1 == 2)
+    Node('PoundAssertExpr', kind='Expr',
+         traits=['Parenthesized'],
+         children=[
+             Child('PoundAssert', kind='PoundAssertToken'),
+             Child('LeftParen', kind='LeftParenToken'),
+             Child('Condition', kind='Expr',
+                   description='The assertion condition.'),
+             Child('Comma', kind='CommaToken', is_optional=True,
+                   description='The command after the assertion condition.'),
+             Child('Message', kind='StringLiteralExpr', is_optional=True,
+                   description='The assertion message.'),
+             Child('RightParen', kind='RightParenToken'),
+         ]),
```

Comment:
Could you add some #assert tests in round_trip_parse_gen.swift to ensure they are constructed as syntax nodes correctly?

Reply:
Missing tags indicates unknown syntax kinds, which unfortunately means these syntax nodes are not constructed properly. You can use command `$./swift-macosx-x86_64/bin/swift-syntax-test -input-source-filename /tmp/t.swift -parse-gen -print-node-kind -print-trivial-node-kind` to print all unknown kinds and diagnose what's wrong with it.

Reply:
Thanks, this helped me figure it out. The problem was that the test runs without "enable-experimental-pound-assert" so the parser bails out before parsing the whole statement, which confuses the syntax thing. I fixed it by making the parser parse the whole statement before bailing out.